src/controllers/roles.js

```diff
@@ -0,0 +1,38 @@
+import Responses from '../utils/response';
+import db from '../models';
+import UserServices from '../services/User.service';
+
+/**
+ * Class for roles related operations
+ */
+class Role {
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} responses
+   */
+  async changeRole(req, res) {
+    const user = await UserServices.findUserByEmail(req.body.email);
```

Comment:
Why `findUserByEmail` and not `findUserById `?

Reply:
The story requirement is that you change a user's role based on their email.

Reply:
That's ok